### PR TITLE
fix: clean and prepare couchdb-sync

### PR DESCRIPTION
**Pull request by obdulia-losantos**
- Improve and clean documentation.
- Introduce *i18n* in messages.
- Remove kernel project fixture for tests, use `artefacts` endpoint to create kernel dependencies.
- Introduce `Schema` model to keep the relationship between the mobile schema and the documents submission to Kernel (remove name dependency with kernel mapping).

### CODE REVIEW

File: aether-couchdb-sync-module/aether/sync/api/views.py [hunk 1]
```diff
@@ -117,13 +164,11 @@ def signin(request):
...
-        return Response('Creating couchdb db failed', status.HTTP_500_INTERNAL_SERVER_ERROR)
+        return Response(_('Creating couchdb db failed'), status.HTTP_500_INTERNAL_SERVER_ERROR)
 
     payload = {
         'username': couchdb_config['username'],
         'password': couchdb_config['password'],
-        # Send the url of the couchdb device-db for replication
-        # 'url': 'http://localhost:8667/_couchdb/' + device_db.db_name
         'db': device_db.db_name,
         'url': request.build_absolute_uri('/_couchdb/' + device_db.db_name),
```
Comment (obdulia-losantos) [inline aether-couchdb-sync-module/aether/sync/api/views.py:173]: This is an alias created in `nginx`, I wonder if this is a good idea or if we should return the `settings.COUCHDB_URL` value instead? :thinking:

**Review by admbtlr — DISMISSED** [review]
Thank you! (I haven't actually tested this, but the code looks great....)

**Comment by obdulia-losantos** [comment]
@adamvert could you review this again? I removed the entities clean up in tests, it's failing in Travis but not locally, I will try to fix it in another PR :crossed_fingers: 
Thanks!

**Comment by obdulia-losantos** [comment]
@adamvert @quesebifurcan do not forget this PR :pray:

**Comment by admbtlr** [comment]
I haven't! I'd like to test it locally, which might take me a while to set up....

**Comment by obdulia-losantos** [comment]
:dancer:

**Comment by admbtlr** [comment]
Still testing...

**Comment by obdulia-losantos** [comment]
_still breaking_

**Comment by admbtlr** [comment]
But now we have the problem that the proxied request still includes the `/_couchdb` in the path, so `http://sync.aether.local/_couchdb/_session` proxies to `http://localhost:5984/_couchdb/_session`, which returns an error.

**Comment by obdulia-losantos** [comment]
Should we try this https://github.com/eHealthAfrica/sense-hat/blob/development/build_scripts/nginx.conf#L34-L37 ?

**Comment by obdulia-losantos** [comment]
image

**Review by shawnsarwar — COMMENTED** [review]
@adamvert What's the status of your previous comment? Can we move this forward?

**Review by admbtlr — APPROVED** [review]

**Comment by admbtlr** [comment]
Works fine now - @obdulia-losantos we probably need to release this PR to get sync working for the Microcensus server, don't we?

**Comment by obdulia-losantos** [comment]
Yes, we also need to create the related data in kernel, like the project, schemas, mappings...

**Comment by admbtlr** [comment]
Right, so let's get this merged and then we can make a 0.9.2 release

**Pull request merged**